### ISSUE TITLE
[Visual Editor] SetProperties does not correctly display object types

### ISSUE THREAD

**Issue by a-b-r-o-w-n** [microsoft/BotFramework-Composer#2309]
![image](https://user-images.githubusercontent.com/3619060/77091479-db7b8300-69c5-11ea-96ab-257b605e305c.png)

**Comment by yeze322** [comment]
What's our expected appearance of an object?
To show its full value or just tell user it's an Object `[Object]`?

**Comment by a-b-r-o-w-n** [comment]
@sangwoohaan do you have guidance here? It's possible for users to add values that are objects with any types in them and arrays with any types in them.

ex:
```
{
  "user": { "address": { "city": "Bellevue" } },
  "places": [{ "name": "Office" }, { "name": "Home" }]
}
```

**Comment by sangwoohaan** [comment]
If values are objects or arrays they should display the actual values, but truncate if it doesn't fit in one line.
![Group 720](https://user-images.githubusercontent.com/50963385/77206243-ba915b80-6ab3-11ea-9259-b68590f58f8d.png)
Something like this

**Comment by yeze322** [comment]
addressed in #2335
